metafix/src/main/java/org/metafacture/metafix/FixMethod.java

```diff
@@ -662,8 +663,16 @@ public void apply(final Metafix metafix, final Record record, final List<String>
         public void apply(final Metafix metafix, final Record record, final List<String> params, final Map<String, String> options) {
             record.transform(params.get(0), s -> s.toUpperCase());
         }
+    },
+    uri_encode {
+        @Override
+        public void apply(final Metafix metafix, final Record record, final List<String> params, final Map<String, String> options) {
+            URL_ENCODER.setPlusForSpace(false);
```

Comment:
Preferrably, we would expose this as an option `plus_for_space` (along with `safe_chars`). But I don't think it's necessarily a blocker.

Reply:
Agreed - we can, if needed, easily enhance to use options.

Reply:
Okay, but I'll unresolve it as a reminder.